<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMultiCopy</class>
 <widget class="QDialog" name="QMultiCopy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>类型</string>
   </property>
   <widget class="QRadioButton" name="Overlap">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>18</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>重叠型</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="Interval">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>18</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>间隔型</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>X方向属性</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X方向数量：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Column">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X方向间隔：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Column_Interval">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>800</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Y方向属性</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y方向数量：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Row">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y方向间隔：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Row_Interval">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>480</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="if_Increse_Address">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>是否地址递增</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Increse_Right">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>地址右增</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="Increse_Down">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>地址下增</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>地址递增间隔：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Increse_Interval">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
